tests/framework/pkg/killserver/killserver.go

```diff
@@ -0,0 +1,43 @@
+package killserver
```

Comment:
So I don't have a strong opinion on this, but does this need to be in `pkg` rather than `extensions/codecoverage/killserver`, what are your thoughts on that, do you think this is something generic to be in `pkg` dir?

Reply:
Yes this really outside of the extensions, and more of system level thing I think it belongs pkg for sure